redis/commands/search/aggregation.py

```diff
@@ -1,8 +1,18 @@
+from enum import Enum
 from typing import List, Union
 
 FIELDNAME = object()
 
 
+class Scorers(Enum):
+    TFIDF = "TFIDF"
```

Comment:
https://redis.io/docs/latest/develop/interact/search-and-query/advanced-concepts/scoring/

available according to docs